R/reportingengine-env.R

```diff
@@ -18,6 +18,9 @@ reEnv$defaultStairstep <- TRUE
 reEnv$defaultWatermarkMessage <- "preliminary analysis"
 reEnv$defaultSimulationSetDescriptor <- ""
 
+reEnv$maximalParametersPerSensitivityPlot <- 15

```

Comment:
@pchelle Didn't we say 25? I suggest to set it to 25 per default

Reply:
We definitely did. Sorry about the typo.
I am going to fix that.